src/test/robot/api/storage/create_and_read_events.robot

```diff
@@ -0,0 +1,90 @@
+*** Settings ***
+Force Tags  create_and_read_events
+Library  OperatingSystem
+Library  CSVLibrary
+Test Setup  Start Containers
+Test Teardown  Stop Containers
+
+*** Variables ***
+${MONGOOSE_IMAGE_NAME} =  emcmongoose/mongoose-storage-driver-pravega
+${MONGOOSE_IMAGE_VERSION} =  testing
+${MONGOOSE_CONTAINER_NAME} =  mongoose-storage-driver-pravega
+${MONGOOSE_CONTAINER_DATA_DIR} =  /data
+
+${PRAVEGA_IMAGE_NAME} =  pravega/pravega
+${PRAVEGA_IMAGE_VERSION} =  latest
+${PRAVEGA_CONTAINER_NAME} =  pravega_standalone
+${PRAVEGA_HOST_IP} =  HOST_IP=127.0.0.1
+${PRAVEGA_CONTROLLER_PORT} =  9090
+${PRAVEGA_SEGMENT_STORE_PORT} =  12345
+${PRAVEGA_RUNNING_MODE} =  standalone
+
+${LOG_DIR} =  build/log
+${DATA_DIR} =  src/test/robot/api/storage/data
+*** Test Cases ***
+Create and read Events Test
+    ${step_id} =  Set Variable  create_and_read_events_test
+    ${host_working_dir} =  Get Environment Variable  HOST_WORKING_DIR
+    Remove Directory  ${LOG_DIR}/${step_id}  recursive=True
+    ${args} =  Catenate  SEPARATOR= \\\n\t
+    ...  --load-step-id=${step_id}
+    ...  --storage-net-node-port=${PRAVEGA_CONTROLLER_PORT}
+    ...  --load-op-recycle
+    ...  --run-scenario=${MONGOOSE_CONTAINER_DATA_DIR}/${step_id}.js
+    ${std_out} =  Execute Mongoose Scenario  ${args}
+    Log  ${std_out}
+    Validate Metrics Total Log File  ${step_id}  READ  5  0  5242880
+
+*** Keyword ***
+Execute Mongoose Scenario
```

Comment:
the keywords here and below are copy-paste product
please consider refactoring to reuse the keywords as was done in s3 driver, for example:
https://github.com/emc-mongoose/mongoose-storage-driver-s3/blob/master/src/test/robot/lib/MongooseContainer.robot
https://github.com/emc-mongoose/mongoose-storage-driver-s3/blob/master/src/test/robot/lib/Validation.py

otherwise, please put all tests in one robot file (easier and faster way)